graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/types/output/CDPEvent.java

```diff
@@ -85,8 +85,7 @@ public CDPProfile getProfile(final DataFetchingEnvironment environment) {
     @GraphQLPrettify
```

Comment:
Use accessor without get instead of @GraphQLPrettify plz to be uniform across the app